lib/osf-components/addon/components/node-card/template.hbs

```diff
@@ -80,7 +80,6 @@
                         {{/if}}
                     {{/if}}
 
-                    {{node-card/node-icon category=@node.category}}
```

Comment:
This will remove it from regular nodes whiiiiich we may not be using anymore but I can't recall for sure. Maybe check `@node.is_registration` or similar and hide it if it's a registration.